Shared/Article Rendering/ArticleRenderer.swift

```diff
@@ -375,40 +388,6 @@ private extension ArticleRenderer {
 		return s
 	}
 	
-	#else
-	
-	func renderHTML(withBody body: String) -> String {
```

Comment:
Is it ok to delete this ios block? As far as I could tell the only difference is the lack of `mouseenter`/`mouseleave` listeners. Should be ok to include these. iOS does not support these events so they should be ignored.

Reply:
Yeah, delete that.